scripts/cc-cli/src/utils/perbill.ts

```diff
@@ -0,0 +1,7 @@
+export function perbillFromPercent(value: number) {
+  return Math.floor(value * 10_000_000);
+}
+
+export function percentFromPerbill(value: number) {
+  return value / 10_000_000;
+}
```

Comment:
QE note: need to inspect where these 2 functions are used and what are the accepted boundary values.